docs/modules/commons/pages/vividus-steps.adoc

```diff
@@ -173,7 +173,7 @@ The created file will be removed upon test run completion.
 
 [source,gherkin]
 ----
-When I create temporary file with name '$name' and content '$content' and put path to '$scopes' variable with name '$pathVariable'
+When I create temporary file with name `$name` and content `$content` and put path to $scopes variable `$pathVariable`
```

Comment:
we need to define explicitly that it's a name of the variable, not the variable reference
```suggestion
When I create temporary file with name `$name` and content `$content` and put path to $scopes variable `$variableName`
```

Reply:
btw the step syntax was updated, but here it's not